blocks/init/src/Blocks/custom/paragraph/manifest.json

```diff
@@ -43,7 +44,7 @@
 		"paragraphAlign": [
 			{
 				"variable": {
-					"block-paragraph-align": "%value%"
+					"paragraph-align": "%value%"
```

Comment:
This looks like a BC break, no? At the minimum, the SCSS files would be affected (variables)?

Reply:
Don't think it's a breaking change because it would only affect newly created projects 😉

Guess you could pull this into your existing project, but you would need to customize it anyways.